pkg/state/state.go

```diff
@@ -2709,17 +2721,33 @@ func (st *HelmState) flagsForDiff(helm helmexec.Interface, release *ReleaseSpec,
 	}
 	flags = st.appendPostRenderFlags(flags, release, postRenderer)
 
-	var postRendererArgs []string
+	postRendererArgs := []string{}
 	if opt != nil {
 		postRendererArgs = opt.PostRendererArgs
 	}
 	flags = st.appendPostRenderArgsFlags(flags, release, postRendererArgs)
 
+	suppressOutputLineRegex := []string{}
+	if opt != nil {
+		suppressOutputLineRegex = opt.SuppressOutputLineRegex
+	}
+	if len(suppressOutputLineRegex) > 0 || len(st.HelmDefaults.SuppressOutputLineRegex) > 0 || len(release.SuppressOutputLineRegex) > 0 {
+		diffVersion, err := helmexec.GetPluginVersion("diff", settings.PluginsDirectory)
+		if err != nil {
+			return nil, nil, err
+		}
+		dv, _ := semver.NewVersion("v3.9.0")
+
+		if diffVersion.LessThan(dv) {
+			return nil, nil, fmt.Errorf("suppressOutputLineRegex is not supported by helm-diff plugin version %s, please use at least v3.9.0", diffVersion)
```

Comment:
I think a warning here is sufficient; it shouldn't affect the subsequent execution.

Reply:
same handling as `insecureSkipTLSVerify`.

Reply:
we should notify user quickly if he/she wanna use the feature. in order to reduce the time to find the reason. @xiaomudk WDYT?